docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,34 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
+| TRACKER        | Broadcasts GPS position packets as priority.                                                                             | Tracking the location of individuals or assets, especially in scenarios where timely and efficient location updates are critical.      |
+| LOST_AND_FOUND | Broadcasts location to default channel regularly for recovery efforts.                                                   | Used for recovery efforts of a lost device.                                                                                            |
+| SENSOR         | Broadcasts priority telemetry packets, optimized for sensor data collection.                                             | Deploying in scenarios where gathering environmental or other sensor data is crucial, with efficient power usage and frequent updates. |
+| TAK            | Optimized for ATAK system communication, reduces routine broadcasts.                                                     | Integration with ATAK systems for communication in tactical or coordinated operations.                                                 |
+| REPEATER       | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Not visible in Nodes list. | Best positioned in strategic locations to maximize the network's overall coverage not shown in topology.                               |
+| ROUTER         | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Visible in Nodes list.     | Best positioned in strategic locations to maximize the network's overall coverage shown in topology.                                   |
+| ROUTER_CLIENT  | Combination of both ROUTER and CLIENT.                                                                                   | Devices in a strategic position for priority routing that need to also serve as a standard CLIENT.                                     |                            
+
+#### Role Comparison
+| Device Role    | Bluetooth/Wi-Fi Enabled | Screen Enabled | Power Consumption | Retransmit | Prioritized Routing | Visible in Nodes List |
+| -------------- | ----------------------- | -------------- | ----------------- | ---------- | ------------------- | --------------------- |
+| CLIENT         | Yes                     | Yes            | Regular           | Yes        | No                  | Yes                   |
+| CLIENT_MUTE    | Yes                     | Yes            | Lowest            | No         | No                  | Yes                   |
+| CLIENT_HIDDEN  | Yes                     | Yes            | Lowest            | Known only | No                  | Yes                   |
```

Comment:
CLIENT_HIDDEN
- Retransmit is Local Only
- Visible in Nodes List, I haven't tested but I assume NO.  (config.device.node_info_broadcast_secs = UINT32_MAX)